os/pm/pm_metrics.c

```diff
@@ -23,6 +23,32 @@
 #include "pm_metrics.h"
 #include "pm.h"
 
+struct pm_metric_global_s g_pm_metrics_globals;
+
+static void pm_print_metrics(void)
+{
+	double suspend_duration;
+	double resume_duration;
+	double suspend_percent;
+	int index;
+	static const char *state_name[] = {"PM_NORMAL", "PM_IDLE", "PM_STANDBY", "PM_SLEEP", NULL};\
+	pmdbg("\n\n");
+	pmdbg("          DOMAIN                | SUSPEND TIME (msec) | SUSPEND TIME (%%) | SUSPEND COUNTS \n");
+	pmdbg("--------------------------------|---------------------|------------------|----------------\n");
+	for (index = 0; (index < CONFIG_PM_NDOMAINS) && pm_domain_map[index]; index++) {
+		suspend_duration = (double)TICK2MSEC(g_pm_metrics_globals.suspend_ticks.ticks[index]);
+		resume_duration = (double)TICK2MSEC(g_pm_metrics_globals.resume_ticks.ticks[index]);
```

Comment:
We know the total time, and it seems that g_pm_metrics_globals.resume_ticks is only necessary to calculate the suspension time ratio.

How about calculating the ratio with just the total time and the suspended time?